arches_component_lab/src/arches_component_lab/widgets/types.ts

```diff
@@ -2,11 +2,6 @@ import {
     CONFIGURE,
     EDIT,
     VIEW,
-} from "@/arches_component_lab/widgets/constants";
+} from "@/arches_component_lab/widgets/constants.ts";
 
 export type WidgetMode = typeof CONFIGURE | typeof EDIT | typeof VIEW;
-
-export interface URLDatatype {
-    url: string;
-    url_label: string;
-}
```

Comment:
Same, used in lingo, however, I expect it's OK to remove this and just mint types for all the datatype in one fell swoop somewhere.

Reply:
++